src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -328,7 +328,7 @@ internal static bool IsCompletedAction(this EditorActionEventArgs e, ImeAction?
 			return
 				actionId == ImeAction.Done ||
 				actionId == currentInputImeFlag ||
-				(actionId == ImeAction.ImeNull && evt?.KeyCode == Keycode.Enter && evt?.Action == KeyEventActions.Up);
+				(actionId == ImeAction.ImeNull && evt?.KeyCode == Keycode.Enter && evt?.Action == KeyEventActions.Down);
```

Comment:
When using a keyboard and pressing Enter, to trigger the `ReturnCommand` we check that the key was Enter and that the action was Up. However, in this [commit](https://github.com/dotnet/maui/commit/0203472ea078066e6957c07d90b94f30418c7198) (Don't set editor action to handled), the EditorAction event is changed, so we will never capture key ups.

Possible options are, apply changes to the EditorAction event to capture key ups or, change the validation to key down.